docs/options.md

```diff
@@ -113,6 +115,9 @@ CIBW_SKIP: cp36-manylinux*
 # Only build on Python 3 and skip 32-bit builds
 CIBW_BUILD: cp3?-*
 CIBW_SKIP: "*-win32 *-manylinux_i686"
+
+# Only build PyPy and CPython 3
+CIBW_BUILD: pp*-* cp3?-*
```

Comment:
Why `pp*-*` not `pp*`?

Reply:
Because ... well, because the PyPy tags were changed a few times, and because above it also says `cp3?-*` rather than `cp3*`; no real further reason. Do you think `pp*` is more clear?

Reply:
Such writing suggest that there are some tags starting with `pp` without `-`.

Reply:
Can you not say the same about `cp3?-*`, then, that it suggests there are some tags with `cp3*` that are not `cp3?-*`?

Reply:
For me there is a big difference between one character match`?` and wildcard `*`. Maybe this example should show pypy3 and cPython3.

Btw. see line 114 This Python should be renamed to cPython?